homeassistant/components/workday/binary_sensor.py

```diff
@@ -43,8 +43,9 @@
 def valid_country(value: Any) -> str:
     """Validate that the given country is supported."""
     value = cv.string(value)
-    all_supported_countries = holidays.list_supported_countries()
-
+    all_supported_countries = (
+        holidays.list_supported_countries() | holidays.list_supported_financial()
```

Comment:
I feel like this is misusing the existing `country` configuration option. We should not mix up these values.